Automattic/wp-calypso
# Remove Unused Jetpack Pricing Intro Banner Logic

#### Changes proposed in this Pull Request

* Remove old `IntroPricingBanner`, rename `IntroPricingBannerV2` to `IntroPricingBanner`
* Remove `jetpack/pricing-page-v2-banner` config flag and associated banner-switching logic

#### Testing instructions

1. Navigate to each page on the branch:
 - **(Calypso Green)** Jetpack Pricing `jetpack.cloud.localhost:3000/pricing`
 - **(Calypso Blue)** WordPress.com Plans `calypso.localhost:3000/plans/:siteSlug`
 - **(Calypso Blue)** Jetpack Connect Store `calypso.localhost:3000/jetpack/connect/store`
2. Verify that the page is the same as `production`.

Related to #59129
